src/main/java/tutorial/Game.java

```diff
@@ -23,18 +26,22 @@ void initialiseGame(int numberOfDiscs) {
     void solve() {
         while(!isComplete()){
             for (Move move:moves) {
+                //parameter type?
                 moveDisc(move.getFromPosition(), move.getToPosition());
+                //push this into moveDisc
                 if(isComplete()) {
                     break;
                 }
             }
         }
     }
 
+    //is move actually a method on Disc?
     void moveDisc(int fromTower, int toTower) {
         Tower startingTower = towersOfHanoi.get(fromTower);
         Tower finishingTower = towersOfHanoi.get(toTower);
 
+        //try to avoid testing negative conditions
         if (!moveIsIllegal(fromTower, toTower)) {
```

Comment:
"not move is illegal" fries my head - it would be more legible if expressed as a positive rather than a double-negative.